### PR TITLE
508

### PR DESCRIPTION
**Pull request by mleeBoeing**
Notification should describe how many results are displaying on the field and now talks about # of results.

### CODE REVIEW

File: app/js/components/discovery/index.jsx [hunk 1]
```diff
@@ -446,13 +446,22 @@ var Discovery = React.createClass({
...
+        let listingResults;
 
         if (!this._searching) {
             results = this.state.searchResults.length > 0 ?
                 ListingTile.fromArray(this.state.searchResults) :
                 <h3 className="col-xs-12">No results found.</h3>;
         }
 
+        if (this.state.searchResults.length != 0) {
+            listingResults = <div aira-label={'Showing (this.state.searchResults.length) out of (DiscoveryPageStore.getTotalSearchResults()) listings'}>
```
Comment (blynch11p) [inline app/js/components/discovery/index.jsx]: aria, not aira.

Comment (blynch11p) [inline app/js/components/discovery/index.jsx]: The variables (this.state.searchResults.length) are showing exactly as typed in the source after fixing the typo.  You might have to build the text elsewhere, as a variable, to put in the html.

Comment (lsemesky) [inline app/js/components/discovery/index.jsx]: try using the jsx syntax by surrounding variables (or computed statements) in brakets like 'Showing {this.state.searchResults.length} ....'  (I think that will work- play around with it if not)

File: app/js/components/discovery/index.jsx [hunk 2]
```diff
@@ -446,13 +446,22 @@ var Discovery = React.createClass({
...
+                Showing {this.state.searchResults.length} out of {DiscoveryPageStore.getTotalSearchResults()} listings.
+            </div>
+        } else {
+            listingResults = null;
```
Comment (lsemesky) [inline app/js/components/discovery/index.jsx]: if you initialize listingResults to null you don't need the else

**Comment by blynch11p** [comment]
:+1:

**Comment by blynch11p** [comment]
:+1:

File: app/js/components/discovery/index.jsx [hunk 3]
```diff
@@ -446,13 +446,22 @@ var Discovery = React.createClass({
...
         var results = '';
+        let listingResults;
 
         if (!this._searching) {
             results = this.state.searchResults.length > 0 ?
                 ListingTile.fromArray(this.state.searchResults) :
                 <h3 className="col-xs-12">No results found.</h3>;
         }
 
+        if (this.state.searchResults.length != 0) {
```
Comment (mleeBoeing) [inline app/js/components/discovery/index.jsx]: :+1:. Will use this from now on

**Pull request merged**